### PR TITLE
EN: more check spelling

### PR DESCRIPTION
**Pull request by alama**

### CODE REVIEW

File: Dialogue/Mid-Mission/co_022500.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
 name01#0,"""Brigitta"""
 skit01#0,"""Emergency contact!<br>Darker reactions surrounding the Campship!<br>It's being pulled by an abnormal gravity source!"""
-skit02#0,"""Confirming transfer of Teleporter coordinates! Due to unforeseen circumstances, pay close attention to your surrounds."""
+skit02#0,"""Confirming transfer of teleporter coordinates! Due to unforeseen circumstances, pay close attention to your surrounds."""
```
Comment (marrub--) [inline Dialogue/Mid-Mission/co_022500.csv:3]: This change is fine

File: Dialogue/co_040039.csv [hunk 2]
```diff
@@ -2,7 +2,7 @@ name01#0,"""Xiera"""
 name01#1,"""Xiera"""
 text01#0,"""<%me>, good work！<br>Xiera, your exclusive operator, here！"""
 text02#0,"""A new piece of the storyboard has appeared!<br>Please make your way to the bridge!"""
-text03#0,"""You can reach the bridge by using the<br>Teleporter on the second floor of the gate area.<br>I'll be awaiting your arrival!"""
+text03#0,"""You can reach the bridge by using the<br>teleporter on the second floor of the gate area.<br>I'll be awaiting your arrival!"""
```
Comment (marrub--) [inline Dialogue/co_040039.csv:5]: This one is fine and doesn't need to be capitalized

File: Misc/actor_name.csv [hunk 3]
```diff
@@ -816,8 +816,8 @@ Npc_162#1,"""Casino Area"""
 Npc_192#2,"""Monument Plaza"""
 Npc_186#1,"""Upper Gate Area"""
 Npc_187#1,"""dummy"""
-Npc_150#1,"""Gate Area: Near Central Teleporter"""
-Npc_151#1,"""Gate Area: Near Central Teleporter"""
+Npc_150#1,"""Gate Area: Near Central teleporter"""
+Npc_151#1,"""Gate Area: Near Central teleporter"""
```
Comment (marrub--) [inline Misc/actor_name.csv]: All instances of "teleporter" in this file need to be capitalized

File: Tutorial/Arks Road/Talk/ar_talk_or011300.csv [hunk 4]
```diff
@@ -1,4 +1,4 @@
 name01#0,"""Astarte"""
 text01#0,"""Your current goal is..."""
-11_voice_shop_414#0,"""To meet <c bb0000>Zeig inside of the Block Teleporter<c>."""
+11_voice_shop_414#0,"""To meet <c bb0000>Zeig inside of the Block teleporter<c>."""
```
Comment (marrub--) [inline Tutorial/Arks Road/Talk/ar_talk_or011300.csv]: This one is also a title and needs to be capitalized.

File: UI/ui_mainmode.csv [hunk 5]
```diff
@@ -3140,7 +3140,7 @@ SynchronizeMyParty#0,"""Your party"""
 SynchronizeMemberName#0,"""Member Name"""
 SynchronizeStageClear#0,"""Cleared Stage"""
 SynchronizeQuestName#0,"""Quest name"""
-SynchronizeBossTitle#0,"""Teleporter"""
+SynchronizeBossTitle#0,"""teleporter"""
```
Comment (marrub--) [inline UI/ui_mainmode.csv]: All changed instances of "teleporter" in this file need to be capitalized, the unchanged ones are fine

**Review by marrub-- — CHANGES_REQUESTED** [review]
Looks good, but "teleporter" needs to be fixed in a few places.

File: UI/ui_quest.csv [hunk 6]
```diff
@@ -103,7 +103,7 @@ Q_Event001_01_MES#0,"""The ARKS are preparing a large-scale operation in respons
 Q_Event001_02_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Naberian tundra."""
 Q_Event001_03_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Naberian ruins."""
 Q_Event001_04_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Amduscian caverns."""
-Q_Event001_05_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Amduscian floating continent."""
+Q_Event001_05_MES#0,"""The ARKS are preparing a large-scale operation in response to enemy activity in the Amduscian Skyscape."""
```
Comment (marrub--) [inline UI/ui_quest.csv]: This shouldn't be capitalized (as it's an area name not a title)

File: UI/ui_quest.csv [hunk 7]
```diff
@@ -173,7 +173,7 @@ Q_TwiceEvent001_01_MES#0,"""The ARKS are preparing multiple large-scale operatio
 Q_TwiceEvent001_02_MES#0,"""The ARKS are preparing multiple large-scale operations in response to enemy activity in the Naberian tundra."""
 Q_TwiceEvent001_03_MES#0,"""The ARKS are preparing multiple large-scale operations in response to enemy activity in the Naberian ruins."""
 Q_TwiceEvent001_04_MES#0,"""The ARKS are preparing multiple large-scale operations in response to enemy activity in the Amduscian caverns."""
-Q_TwiceEvent001_05_MES#0,"""The ARKS are preparing multiple large-scale operations in response to enemy activity on the Amduscian floating continents."""
+Q_TwiceEvent001_05_MES#0,"""The ARKS are preparing multiple large-scale operations in response to enemy activity on the Amduscian Skyscape."""
```
Comment (marrub--) [inline UI/ui_quest.csv:176]: ditto

File: Dialogue/Mid-Mission/co_011257.csv [hunk 8]
```diff
@@ -11,5 +11,5 @@ skit02#0,"""It definitely makes the stage harder, but<br>if you complete the cha
 skit03#0,"""Not only that, but completing the challenges<br>will also increase the numbers of items you get to<br>keep at the end of the simulation!"""
 skit04#0,"""Hello! Thank you for coming. My name is Karin!"""
 skit05#0,"""You've been given the chance to take part in this extreme quest.<br>We'll be seeing a lot of each other!"""
-skit06#0,"""Ohohoho!<br>Please proceed to the teleporter.<br>This is where the fun begins!"""
+skit06#0,"""Ohohoho!<br>Please proceed to the Teleporter.<br>This is where the fun begins!"""
```
Comment (marrub--) [inline Dialogue/Mid-Mission/co_011257.csv]: This one should also be lowercase

File: Dialogue/Mid-Mission/incident_common.csv [hunk 9]
```diff
@@ -140,7 +140,7 @@ FloororderPartDamageR#0,"""Destroy$(2)$(1) <br>parts$(0)!"""
 FloorOrderNoDamageR#0,"""Don't take damage$(0)$(1)!"""
 FloorOrderNoRecoverR#0,"""Don't heal$(0)$(1)!"""
 FloorOrderNoDeadR#0,"""Don't die$(0)$(1)!"""
-BonusmapAppearanceMsg#0,"""A teleporter to a Parallel Area has appeared in $(0)."""
+BonusmapAppearanceMsg#0,"""A Teleporter to a Parallel Area has appeared in $(0)."""
```
Comment (marrub--) [inline Dialogue/Mid-Mission/incident_common.csv]: This should be lowercase

File: Tutorial/Arks Road/Give Card/ar_givecard_ca0113.csv [hunk 10]
```diff
@@ -1,8 +1,8 @@
 name01#0,"""Astarte"""
 11_voice_shop_412#0,"""You have met the requirements for your next stamp card. Here you go."""
 text02#0,"""This card focuses on Crafting."""
-text03#0,"""You are to meet Zieg who is just beyond the teleporters in the shop area."""
-text04#0,"""From the gate area, these teleporters are located beside the Quest Counters."""
-text05#0,"""From the shop area, these are the teleporters against the outside walls."""
+text03#0,"""You are to meet Zieg who is just beyond the Teleporters in the shop area."""
+text04#0,"""From the gate area, these Teleporters are located beside the Quest Counters."""
+text05#0,"""From the shop area, these are the Teleporters against the outside walls."""
```
Comment (marrub--) [inline Tutorial/Arks Road/Give Card/ar_givecard_ca0113.csv]: These should be lowercase

File: Tutorial/Dialogue/co_010008.csv [hunk 11]
```diff
@@ -4,8 +4,8 @@ tutorial01#0,"""A new client order is available for Marlu.<br><br>Marlu can be f
 tutorial02#0,"""A new client order is available for Ohza.<br><br>Ohza can be found in the gate area, Central Hall."""
 tutorial03#0,"""A new client order is available for Lisa.<br><br>Lisa can be found in the gate area, Central Hall."""
 tutorial04#0,"""A new client order is available for Azanami.<br><br>Azanami can be found in the gate area, Central Hall."""
-tutorial05#0,"""A new client order is available for Saga.<br><br>Saga can be found in the gate area, by the central teleporter."""
-tutorial06#0,"""A new client order is available for Katori.<br><br>Katori can be found in the gate area, by the central teleporter."""
+tutorial05#0,"""A new client order is available for Saga.<br><br>Saga can be found in the gate area, by the central Teleporter."""
+tutorial06#0,"""A new client order is available for Katori.<br><br>Katori can be found in the gate area, by the central Teleporter."""
```
Comment (marrub--) [inline Tutorial/Dialogue/co_010008.csv]: These should be lowercase

File: Tutorial/Window/wh_tl_co_011169.csv [hunk 12]
```diff
@@ -1,4 +1,4 @@
 name01#0,"""The Gate Area"""
 text01#0,"""This will explain about the ARKS Ship.<br><br>Here is the ARKS ship gate area.<br><br>Here, you can accept quests through the Quest Counter, or level up your character's skill level at the class counter."""
 text02#0,"""Talk to the individual at the Quest Counter and select the Quest you wish to take.<br><br>There are 2 locations for the Quest Counter, but they are both the same."""
-text03#0,"""You can get to the shop area using the teleporter.<br><br>At the shop area, you can purchase weapons and armor units, or change your character's appearance at the salon. Feel free to take a look around at everything."""
+text03#0,"""You can get to the shop area using the Teleporter.<br><br>At the shop area, you can purchase weapons and armor units, or change your character's appearance at the salon. Feel free to take a look around at everything."""
```
Comment (marrub--) [inline Tutorial/Window/wh_tl_co_011169.csv]: This should be lowercase

File: UI/ui_mainmode.csv [hunk 13]
```diff
@@ -3140,20 +3140,20 @@ SynchronizeMyParty#0,"""Your party"""
 SynchronizeMemberName#0,"""Member Name"""
 SynchronizeStageClear#0,"""Cleared Stage"""
 SynchronizeQuestName#0,"""Quest name"""
-SynchronizeBossTitle#0,"""teleporter"""
+SynchronizeBossTitle#0,"""Teleporter"""
 SynchronizeStartTitle#0,"""Start Gate"""
-SynchronizeBossMessage#0,"""This teleporter will automatically activate<br>when all members are present. <br><br>Furthermore, the party leader may depart early<br>with those members who<br>have accessed the teleporter."""
-SynchronizeBossMessage_ex#0,"""This teleporter will automatically activate<br>when all members are present."""
+SynchronizeBossMessage#0,"""This Teleporter will automatically activate<br>when all members are present. <br><br>Furthermore, the party leader may depart early<br>with those members who<br>have accessed the Teleporter."""
+SynchronizeBossMessage_ex#0,"""This Teleporter will automatically activate<br>when all members are present."""
```
Comment (marrub--) [inline UI/ui_mainmode.csv]: These two should be lowercase

**Review by marrub-- — CHANGES_REQUESTED** [review]
Almost there

File: UI/ui_quest.csv (hunk 6 above)
Comment (alama) [reply UI/ui_quest.csv]: `<AlamTaz> ` `Q_TwiceEvent001_05_MES#0,"""The ARKS are preparing multiple large-scale operations in response to enemy activity on the Amduscian Skyscape."""`
`<SynthSy>` much better
`<Kyle873>` 👌

Comment (marrub--) [reply UI/ui_quest.csv]: Reading over Synth's last commit, yeah, it should be capitalized. Just seemed odd because the rest of the areas aren't :P

**Review by marrub-- — APPROVED** [review]

**Pull request merged**